src/Sulu/Bundle/PageBundle/Sitemap/PagesSitemapProvider.php

```diff
@@ -126,7 +126,7 @@ private function generateSitemapUrl(
         string $host,
         string $scheme
     ) {
-        $changed = $contentPage['changed'];
+        $changed = $contentPage['lastModified'] ?? $contentPage['changed'];
```

Comment:
@chirimoya is the new expected behaviour not returning the created instead of changed date of the node?

Reply:
As discussed with @chirimoya to avoid a backwards compatibility break we will keep here the fallback to `changed` like currently ipmlemented. We maybe will change this in future when it is maybe possible in the UI of the publishing to set `update lastmodified timestamp` via a toggler.